### PR TITLE
CCAP-26-Create-privacy-policy-page

### PR DESCRIPTION
**Pull request by rapicastillo**
#### ✅ Completion tasks

- [x] The privacy policy page exists
- [x] The privacy policy contains the content from [this doc](https://docs.google.com/document/d/1AYF90xu_8h28wmLxNZJUEFr1AHMGvhWJrtCXY6cvCRs/edit#heading=h.k13rt2h26pr9)
- [x] All the content on the privacy policy screen is voiced by a screen reader
- [x] All interactive elements (links and email addresses) are navigable using tabbed browsing
- [x] When the privacy policy page loads a `page_load` event fires.Meets acceptance criteria

### CODE REVIEW

**Comment by rapicastillo** [comment]
Just to note that I have added the same keys in messages_es.properties but I didn’t translate them.

**Comment by coltborg** [comment]
> Just to note that I have added the same keys in messages_es.properties but I didn’t translate them.

You don't have to add the keys in `es`, as long as they're in the source file, Transifex can add them to `es`. Though I don't think adding them is a problem, so I think it's fine to keep them?

**Comment by coltborg** [comment]
@rapicastillo oh, but I wouldn't add English into the `es` file. If translations aren't available in Spanish, the default is to use the source keys.

File: src/main/resources/templates/privacy.html [hunk 1]
```diff
@@ -4,55 +4,217 @@
...
-        </div>
-        <div class="spacing-below-35">
-          <a href="#how-we-use" th:text="#{privacy.how-we-use.title}"></a>
+  <main id="content" role="main" class="slab slab--white padding-top-0 spacing-below-0">
+    <section class="slab slab--light-purple slab--hero">
+      <div class="grid">
+        <div class="grid__item">
+          <h1 class="spacing-below-35" th:text="#{privacy.title}"></h1>
+          <p th:text="#{privacy.sub-title}"></p>
+          <a href="/flow/ubi/howThisWorks" th:text="#{privacy.apply-now}"
```
[the remark this answers is not in the document]
Comment (analoo) [reply src/main/resources/templates/privacy.html:13]: The current url is invalid. I'd recommend sending it to getting started and maybe passing the language based on what privacy policy the applicant has selected

**Comment by analoo** [comment]
One note (beyond the scope of this ticket). The anchors that bring you down to the headers are great! But then it's hard to go back up to the table of contents again. It feels like there should be a "back to top link" or that the table of content should stay in view the whole time.

**Comment by enyia21** [comment]
Do we have guidance about how the privacy page should look?

File: src/main/resources/templates/privacy.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (coltborg) [reply src/main/resources/templates/privacy.html:13]: +1

**Review by coltborg — DISMISSED** [review]
Please address the URL location change for the `Apply Now` button.

File: src/main/resources/templates/privacy.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (rapicastillo) [reply src/main/resources/templates/privacy.html:13]: Thank you!

**Review by rapicastillo — COMMENTED** [review]
Made sure we redirect to the onboarding forms and reflect the language!

```
          <a
                  th:href="|/flow/gcc/onboarding-getting-started?lang=${#locale.getLanguage()}|"
                  th:text="#{privacy.apply-now} + ' →'"
             class="button--primary button"></a>
```

File: src/main/resources/templates/privacy.html [hunk 2]
```diff
@@ -4,55 +4,219 @@
...
+  <main id="content" role="main" class="slab slab--white padding-top-0 spacing-below-0">
+    <section class="slab slab--light-purple slab--hero">
+      <div class="grid">
+        <div class="grid__item">
+          <h1 class="spacing-below-35" th:text="#{privacy.title}"></h1>
+          <p th:text="#{privacy.sub-title}"></p>
+          <a
+                  th:href="|/flow/gcc/onboarding-getting-started?lang=${#locale.getLanguage()}|"
+                  th:text="#{privacy.apply-now} + ' →'"
+             class="button--primary button"></a>
```
Comment (rapicastillo) [inline src/main/resources/templates/privacy.html:16]: Applied changes here

**Review by enyia21 — APPROVED** [review]

**Pull request merged**